OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- Roming22 
+- fgiloux
+- ramessesii2
+- bnallapeta
+
+approvers:
```

Comment:
Why two lists of approvers? Shouldn't the second list be reviewers? In that case the list could be extended to Xin and Moemen

Reply:
fixed. And updated the lists of approvers and reviewers according to @bnallapeta 's suggestion